lib/engine/game/g_18_uruguay/game.rb

```diff
@@ -535,6 +545,10 @@ def operating_order
 
           super.reject { |c| c == @rptla }.append(@rptla)
         end
+
+        def ship_capacity(train)
+          SHIP_CAPACITY[train.name.partition('+')[0]]
```

Comment:
What's the call to partition doing here? Isn't `train.name` going to be something like `Ship 3` with no plus sign in it?

Reply:
You are correct this was legacy code, removed

Also added nil check in ship_capacity